recipes/linux-headers/linux-headers.inc

```diff
@@ -4,7 +4,7 @@ inherit kernel-headers
 RECIPE_TYPES = "machine sdk"
 
 require conf/fetch/kernelorg.conf
-SRC_URI = "${KERNELORG_MIRROR}/linux/kernel/v${KERNEL_VERSION_PATCHLEVEL}/linux-${PV}.tar.bz2"
+SRC_URI = "${KERNELORG_MIRROR}/linux/kernel/v${@'${PV}'.split('.')[0]+'.x'}/linux-${PV}.tar.xz"
```

Comment:
More to the point: It doesn't make sense to have an .inc file which is unusable by the current in-tree recipes.